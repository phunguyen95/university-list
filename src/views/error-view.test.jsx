import { render, screen } from "@testing-library/react";
import ErrorView from "./error-view";
import { Router } from "react-router-dom";
import { createMemoryHistory } from "history";

test("renders error view without crashing", () => {
    const history = createMemoryHistory();

  const { asFragment } = render(<Router history={history}><ErrorView /></Router>);

  expect(asFragment()).toMatchSnapshot();
});
