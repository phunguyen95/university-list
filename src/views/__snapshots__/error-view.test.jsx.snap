// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders error view without crashing 1`] = `
<DocumentFragment>
  <section
    class="section is-medium notfound"
  >
    <div
      class="container"
    >
      <div
        class="columns is-vcentered"
      >
        <div
          class="column has-text-centered"
        >
          <h1
            class="title"
          >
            404 Page Not Found
          </h1>
          <p
            class="subtitle"
          >
            An unexpected error has occurred..
          </p>
          <a
            class="button"
            href="/"
          >
            Visit Homepage
          </a>
        </div>
        <div
          class="column has-text-centered"
        >
          <img
            src="https://cellphones.com.vn/sforum/wp-content/uploads/2021/09/404.2.png"
          />
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
